packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -271,13 +203,8 @@ const AccountRecoveryResetPassword = ({
       // required for this check
       const sessionIsVerified = await account.isSessionVerifiedAuthClient();
 
-      // FOLLOW-UP: Functionality not yet available. FXA-7045
-      await account.setLastLogin(Date.now());
-
-      // FOLLOW-UP: Functionality not yet available. FXA-7045
-      notifier.onAccountSignIn(account);
-
-      relier.resetPasswordConfirm = true;
+      // TODO: do we need this? Is integration data the right place for it if so?
```

Comment:
Did we lose a couple operations here? Are we sure these are in the scope of this PR?

Reply:
Heh so I meant to call this out. I was under the impression that in the React version, we're not automatically logging users in when they reset so we don't need this `setLastLogin` call, or `notifier.onAccountSignIn` call. TBH I can't remember exactly why but I do remember @vbudhram and I talking about it a while back. However... I just realized we have that `notifyFirefoxOfLogin` call for the sync flow, and while I'm not redirected to Settings like I am now, it looks like I can access it if I want to after reset.

I'll file a follow up issue regarding this. I could add it back here but those calls weren't doing anything yet.

Reply:
filed: FXA-8144